webhook/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.10 as builder
+
+RUN go get github.com/Masterminds/glide
```

Comment:
Is that we want to include proxy settings ?

```# cd .; git clone https://github.com/Masterminds/glide /go/src/github.com/Masterminds/glide
Cloning into '/go/src/github.com/Masterminds/glide'...
fatal: unable to access 'https://github.com/Masterminds/glide/': Failed to connect to github.com port 443: Connection timed out
package github.com/Masterminds/glide: exit status 128```

Reply:
We cannot predict what proxy configuration will the end users need in their office or lab, right?

Anyway there's no need to set proxy in the Dockerfile in the first place. User can add proxy to docker build command `docker build --build-arg HTTP_PROXY=http://10.20.30.2:1234 .` as described [here](https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg). Also newer Docker versions support setting proxy in JSON config file which works also for building images: https://docs.docker.com/network/proxy/

Reply:
What about adding it in our docker file ? ```docker build --build-arg http_proxy=$(http_proxy) \
		--build-arg HTTP_PROXY=$(HTTP_PROXY) \
		--build-arg https_proxy=$(https_proxy) \
		--build-arg HTTPS_PROXY=$(HTTPS_PROXY) \
		--build-arg no_proxy=$(no_proxy) \
		--build-arg NO_PROXY=$(NO_PROXY) \```

Reply:
change it something like this

```
#!/usr/bin/env bash
set -e

docker build --build-arg http_proxy=${http_proxy} \
                --build-arg HTTP_PROXY=${HTTP_PROXY} \
                --build-arg https_proxy=${https_proxy} \
                --build-arg HTTPS_PROXY=${HTTPS_PROXY} \
                --build-arg no_proxy=${no_proxy} \
                --build-arg NO_PROXY=${NO_PROXY} -t multus-webhook .
```